mavros/src/plugins/waypoint.cpp

```diff
@@ -556,6 +561,7 @@ class WaypointPlugin : public plugin::PluginBase {
 		}
 	}
 
+	// @brief Callback for scheduled waypoint pull
```

Comment:
The comment should on the following format to be accepted on Doxygen:
`//! @brief ....`
or
```
  /** @brief ......
   */
```
The first is better for one line comment though.